perfkitbenchmarker/benchmark_spec.py

```diff
@@ -665,6 +682,14 @@ def Delete(self):
         logging.exception('Got an exception deleting networks. '
                           'Attempting to continue tearing down.')
 
+    if self.vpn_service:
```

Comment:
Why do the vpn attributes need to be reinitialized when they are being deleted?

Reply:
oops - that was a workaround for a pickling issue with partial runs (--run_stage) that's since been resolved. Just calling vpn_service.Delete() now.